django_ninja_crudl/mixins/soft_delete.py

```
@@ -0,0 +1,31 @@
+"""Mixins for soft-delete handling."""
+
+# pyright: reportUnusedParameter=false
+from abc import ABC
+from typing import Generic
+
+from django.db.models import Model, QuerySet
+
+from django_ninja_crudl.types import RequestDetails, TDjangoModel
+
+
+class SoftDeleteMixin(Generic[TDjangoModel], ABC):
```

Ruff (0.14.10)

`Generic[]` should always be the last base class

Move `Generic[]` to the end

(PYI059)